### PR TITLE
Remove options besides python target

### LINKED ISSUE #18: Update compilation parameters to only Python target

### PR DESCRIPTION
**Pull request by nkadonaga**
Resolve #18 

screen shot 2018-04-06 at 2 53 25 pm

- [x] remove extra options
- [x] frontend tests pass
- [x] manually testing works
- [x] remove redundancy of "Allowable Targets" as a label and a ghost option

Before:

screen shot 2018-04-06 at 3 40 03 pm

After:

screen shot 2018-04-06 at 3 24 51 pm

- [ ] rename "Allowable Targets"? doesn't it make more sense to call it "target Python version" or something?

### CODE REVIEW

**Comment by jcruz** [comment]
I think its ok to have the label redundancy but I'd rename it to `'target'` (for example, a user can open the dropdown and forget what the setting is because the open dropdown hides the label). Probably best to rename `'Allowable Targets'`. I vote `'Target Python version'`.

I'd also get rid of the `>=` in the options.

File: src/components/PersistentDrawer.js [hunk 1]
```diff
@@ -108,26 +108,13 @@ const styles = theme => ({
 });
 
 const optionKeyLabels = {
-  target: 'Allowable Target',
-  strict: 'Enforce code cleanliness standards',
-  minify: 'Reduce size of Compiled Python',
-  line_numbers: 'Show line numbers',
-  keep_lines: 'Keep line numbers',
-  no_tco: 'No TCO',
+  target: 'Allowable Targets',
```
Comment (mjenrungrot) [inline src/components/PersistentDrawer.js]: Change to just Python version target as @jcruz said.

**Review by mjenrungrot — CHANGES_REQUESTED** [review]

File: src/components/PersistentDrawer.js (hunk 1 above)
Comment (nkadonaga) [reply src/components/PersistentDrawer.js]: Done

**Comment by jcruz** [comment]
Let's change the `'Current latest version'` option to `'Latest version'` to be more concise.

**Review by jcruz — APPROVED** [review]

**Pull request merged**